test/utils/matchers.go

```diff
@@ -2,60 +2,150 @@ package utils
 
 import (
 	"fmt"
+	"net/url"
 
+	"github.com/containers/common/pkg/config"
+	. "github.com/onsi/gomega"
 	"github.com/onsi/gomega/format"
 	"github.com/onsi/gomega/gexec"
+	"github.com/onsi/gomega/matchers"
+	"github.com/onsi/gomega/types"
 )
 
+// HaveActiveService verifies the given service is the active service
+func HaveActiveService(name interface{}) OmegaMatcher {
+	return WithTransform(
+		func(cfg *config.Config) string {
+			return cfg.Engine.ActiveService
+		},
+		Equal(name))
+}
+
+type ServiceMatcher struct {
+	types.GomegaMatcher
+	Name                  interface{}
+	URI                   interface{}
+	Identity              interface{}
+	failureMessage        string
+	negatedFailureMessage string
+}
+
+func VerifyService(name, uri, identity interface{}) OmegaMatcher {
```

Comment:
I like this validation of a given service, but should we add, in CI a generic way to actually test the given connection? Not sure if this already exists but could be helpful for some basic use cases